src/pages/sources/components/AddSourceModal.tsx

```diff
@@ -79,7 +79,7 @@ const AddSourceModal: React.FC<AddSourceModalProps> = ({ source, type, onClose,
       credentials: credentials.map(c => Number(c)),
       hosts: values['hosts'].split(','),
       name: values['name'],
-      port: !isNetwork ? '443' : values['port'] || '22',
+      port: !isNetwork ? values['port'] || '443' : values['port'] || '22',
```

Comment:
so values can always override, even if not `isNetwork`?

if so, probably better to do like... `values['port'] || (isNetwork ? '22' : '443')`

Reply:
fixed